models/address.rb

```diff
@@ -0,0 +1,12 @@
+class Address < ActiveRecord::Base
+
+  belongs_to :user
+
+
+  def address_to_string_no_zip
```

Comment:
Not sure what this method is intended to do. In every scenario, I'd expect it to output `",,"`